src/Countries/France.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class France extends Country
+{
+    public function countryCode(): string
+    {
+        return 'fr';
+    }
+
+    /** @return array<string, CarbonImmutable> */
```

Comment:
this return type is wrong (but anyway, you should not need to specify it here. Inheriting it from the parent is better)